### ISSUE TITLE
Doesn't Work For Some Reason (MSI Cyborg 15 A12V)

### ISSUE THREAD

**Issue by Metercedes** [Metabolix/HackBGRT#149]
There's a part like this in the logs section:

```
2023-10-17 01:39:31.374 | pid 14332 | Enabled NVRAM entry for HackBGRT.
2023-10-17 01:39:31.376 | pid 14332 | Verifying config: boot=MS
2023-10-17 01:39:31.378 | pid 14332 | DetectLoader failed: A:\EFI\HackBGRT\bootmgfw-original.efi not found
2023-10-17 01:39:31.442 | pid 14332 | Completed action 'enable-entry' successfully.

```

In BIOS, I disabled Secure Boot
In Windows I disabled BitLocker
I couldn't see any new driver on 'This PC' section.

SETUP LOG FOR EXTRA INFORMATION:
[setup.log](https://github.com/Metabolix/HackBGRT/files/12922840/setup.log)

![image](https://github.com/Metabolix/HackBGRT/assets/51666146/f312c5b4-fac7-43cf-ba1b-11f5b4df7f23)

**Comment by Metabolix** [comment]
So did you reboot and can you maybe see HackBGRT as a boot option in your "BIOS" (actually Efi setup utility)?

**Comment by Metabolix** [comment]
Thanks for the log: it revealed that BitLocker detection had a bug when multiple partitions are present, and this has been fixed in v2.3.1.

As for your original proplem, please try with the latest release (v2.3.1). Make sure you properly reboot the computer after installing. The newest version also detects whether you have rebooted your computer (which can be tricky if you have Fast Startup a.k.a. Hiberboot enabled).

**Comment by Metabolix** [comment]
Glad to hear that it works now.

You can't change the initial logo which the computer shows before loading any software. It would require firmware modification which is risky and machine-dependent if possible at all. So no fix for that, sorry.
